source/plugin/bukkit-sponge-compare.rst

```diff
@@ -0,0 +1,680 @@
+==========================
+Comparing Bukkit to Sponge
+==========================
+
+While the BukkitAPI is notably different to SpongAPI, there are some Events and classes which have an equivalent. This
+page is intended to provide a (probably incomplete) list to help former Bukkit devs out when first trying SpongeAPI.
+
+.. source of this table: https://gist.github.com/kashike/ecb34eb835383b05792b
+
+.. note::
+  To keep the table small, ``org.bukkit.`` and ``org.spongepowered.api.`` were removed in front of the class names.
+
+====================================================== ===============================================================
+BukkitAPI (``org.bukkit.*``)                            SpongeAPI (``org.spongepowered.api.*``)
+====================================================== ===============================================================
+block.banner.Pattern
+block.banner.PatternType                               data.type.BannerPatternShapes
+block.Banner                                           block.tileentity.Banner
+block.Beacon                                           block.tileentity.carrier.Beacon
+block.Biome                                            world.biome.BiomeType
+block.Block                                            block.BlockState
+block.BlockFace                                        util.Direction
+block.BlockState                                       block.BlockSnapshot
+block.BrewingStand                                     block.tileentity.carrier.BrewingStand
+block.Chest                                            block.tileentity.carrier.Chest
+block.CommandBlock                                     block.tileentity.CommandBlock
+block.ContainerBlock                                   block.tileentity.carrier.TileEntityCarrier
+block.CreatureSpawner                                  block.tileentity.MobSpawner
+block.Dispenser                                        block.tileentity.carrier.Dispenser
+block.DoubleChest
+block.Dropper                                          block.tileentity.carrier.Dropper
+block.Furnace                                          block.tileentity.carrier.Furnace
+block.Hopper                                           block.tileentity.carrier.Hopper
+block.Jukebox                                          block.tileentity.Jukebox
+block.NoteBlock                                        block.tileentity.Note
+block.PistonMoveReaction
```

Comment:
Its for the reaction of a block to being pushed or pulled by a piston, either `RESIST`, `BREAK`, or `MOVE`. Could add a property for it.